docs/user/dashboard/tsvb.asciidoc

```diff
@@ -1,126 +1,165 @@
-  It depends, but most often it's two causes:
-
-  1. If looking at a *Time series* chart with a derivative function, the time interval might be too small.
-    Derivatives require sequential values,
-  2. If looking at anything but *Time series* in TSVB, the cause is probably the *Data timerange mode*.
-    This is controlled by *Panel options > Data timerange mode > Entire time range*. This is because
-    TSVB defaults to showing the *last whole bucket*. If the time range is "last 24 hours", and the
-    current time is 9:41, then TSVB metrics will be only show 10 minutes: from 9:30 to 9:40.
-
-How do I calculate the difference between two series?::
-  TSVB doesn't support math across series, but <<create-panels-with-timelion, Timelion>> does. <<vega, Vega>> can also do this.
-
-How do I view the current vs previous month?::
-  While it's not possible to do math on these two, TSVB supports series with time offsets.
-  Click *Clone Series*, and then choose a new color for the new series. Go toc
-  *Options > Offset series time by* and choose an offset value.
-
-How do I calculate a month over month change?::
-  This is not fully supported in TSVB, but there is a special case that is supported *if* the TSVB
-  time range is set to 3 months or more *and* the time interval is one month. Use the `derivative`
-  to get the absolute monthly change. To convert to a percent, add a `math` function with the formula
-  `params.current / (params.current - params.derivative)`, and then set the formatter to Percent.
-  +
-  For other types of month over month calculations, use <<create-panels-with-timelion, Timelion>> or <<vega, Vega>>.
-
-How do I calculate the duration between start and end of an event?::
-  TSVB can't do this because it requires correlation between different time periods. TSVB requires
-  that the duration is pre-calculated.
+For answers to frequently asked *TSVB* question, review the following. 
+
+[float]
+===== Why is my TSVB visualization missing data?
+
+It depends, but most often there are two causes:
+
+* For *Time series* visualizations with a derivative function, the time interval can be too small. Derivatives require sequential values. 
+
+* For all other *TSVB* visualizations, the cause is probably the *Data timerange mode*, which is controlled by *Panel options > Data timerange mode > Entire time range*.
+By default, *TSVB* displays the last whole bucket. For example, if the time filter is set to *Last 24 hours*, and the
+current time is 9:41, *TSVB* displays only the last 10 minutes &mdash; from 9:30 to 9:40.
+
+[float]
+===== How do I calculate the difference between two data series?
+
+Performing math across data series is unsupported in *TSVB*. To calculate the difference between two data series, use <<create-panels-with-timelion, Timelion>> or <<vega, Vega>>.
+
+[float]
+===== How do I compare the current versus previous month?
+
+While math is unsupported in *TSVB*, series with tie offsets is supported.
```

Comment:
Clarifying this because TSVB does support math, just not this kind of math.

```suggestion
*TSVB* can display two series with time offsets, but it can't perform math across series. To add a time offset:
```